mlos_core/mlos_core/tests/optimizers/optimizer_multiobj_test.py

```diff
@@ -90,7 +89,6 @@ def objective(point: pd.DataFrame) -> pd.DataFrame:
     (all_configs, all_scores, all_contexts) = optimizer.get_observations()
     assert isinstance(all_configs, pd.DataFrame)
     assert isinstance(all_scores, pd.DataFrame)
-    assert all_contexts is None
```

Comment:
Same here.  Can we assert that the contexts are something useful?

Reply:
Same as above, unless we split out the SMAC tests

Reply:
Don't need to split them out, just add a branch in the test for them.
I want to make sure we're testing expected functionality in both directions.
It also makes it more obvious what needs to be done when we extend it elsewhere.

Reply:
I made the associated changes